go/host/host.go

```diff
@@ -226,11 +257,6 @@ func (a *Node) ReceiveTx(tx common.EncryptedTx) {
 	a.txP2PCh <- tx
 }
 
-// DB returns the DB of the node
-func (a *Node) DB() *DB {
```

Comment:
This getter was useless - it was only used within this file, where the field is directly accessible.